dpdata/driver.py

```diff
@@ -81,6 +83,35 @@ def label(self, data: dict) -> dict:
         """
         return NotImplemented
 
+    def minimize(self, data: dict) -> dict:
+        """Minimize the geometry.
+
+        If not implemented, this method calls ASE to minimize.
+
+        Parameters
+        ----------
+        data : dict
+            data with coordinates and atom types
+        
+        Returns
+        -------
+        dict
+            labeled data with minimized coordinates, energies, and forces
+        """
+        from ase.optimize import LBFGS
+        
+        system = dpdata.System(data=data)
+        # list[Atoms]
+        structures = system.to_ase_structure()
+        labeled_system = dpdata.LabeledSystem()
+        for atoms in structures:
+            atoms.calc = self.ase_calculator
+            dyn = LBFGS(atoms, logfile=None)
+            dyn.run(fmax=5e-3)
+            ls = dpdata.LabeledSystem(atoms, fmt="ase/structure", type_map=data['atom_names'])
+            labeled_system.append(ls)
```

Comment:
Does one have any chance of specifying the optimizer and the parameters for optimization?

Reply:
Good point, let me consider how to design the API

Reply:
The API is refactored.